SerhiiBuza/HW6/task1.py

```diff
@@ -0,0 +1,24 @@
+
+
```

Comment:
good job
look at another way

lst = list(range(10))
even_list = [num for num in lst if num != 0 and num % 2 == 0]
odd_list = [num for num in lst if num != 0 and num % 3 == 0]
last_numbers = [num for num in lst if num != 0 and num % 2 != 0 and num % 3 != 0]

print(f"Start list: {lst}")
print(f"Even list: {even_list}")
print(f"Odd list that /3 is: {odd_list}")
print(f"Another number in start list: {last_numbers}")